api/HouseOutput.cs

```diff
@@ -11,15 +19,101 @@ namespace api
  */
 public class HouseOutput : IDeviceOutput
 {
-	public HouseOutput()
+	public HouseOutput(string house_info, string device_info)
 	{
+		_houseInfo = house_info;
+		_deviceInfo = device_info;
 	}
 
 	public bool write(Device dev)
 	{
-		//TODO: Implement writing to the house
+		string json = JsonConvert.SerializeObject(dev);
+		string houseID = dev.ID.HouseID.ToString();
+		string roomID = dev.ID.RoomID.ToString();
+		string deviceID = dev.ID.DeviceID.ToString();
+
+		var _ = writeHelper(json, houseID, roomID, deviceID);
```

Comment:
You will need to add an await here in order to ensure you get the expected response, I believe. Also, the variable name `_` is not terribly descriptive - it would be nice to give it a meaningful name.